packages/wonder-blocks-dropdown/src/components/__tests__/dropdown-core.test.js

```diff
@@ -1,14 +1,12 @@
 // @flow
 import * as React from "react";
-import {fireEvent, render, screen, waitFor} from "@testing-library/react";
-import userEvent from "../../../../../utils/testing/user-event.js";
+import {fireEvent, render, screen} from "@testing-library/react";
```

Comment:
Now that we are not using `react-window` by default, I was able to remove the custom `user-event.js` implementation and remove some extra steps in all the tests inside this file.

Reply:
Yay! Do this mean that we no longer need to wait in the webapp tests for things that use wonder-blocks-dropdown?